### DISCUSSION TITLE
Custom domain - does <username>.github.io still work?

### DISCUSSION THREAD

**Discussion by sylikc** [community/community#49163] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

So, I had a quick question.  I read all the help pages on using a custom domain, but it's unclear to me...

Let's say I had two projects, `projectsite1`, `projectsite2`...

If I enable, say `abcdefg.com` for a custom domain for `projectsite2`, I presume I'm able to access `projectsite2`'s GH Pages by using `abcdefg.com`

However, will my `sylikc.github.io/projectsite1` still work?  or do I _have to_ use `abcdefg.com/projectsite1`?

**Comment by yoannchaudet** [comment]
There are two types of Pages sites (see [here](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites)).

Project and user sites.

If you set a custom domain on your user site, your project sites (that don't have a custom domain themselves) will have the new URL. For instance `sylikc.github.io/projectsite1` would become `abcdefg.com/projectsite1`. The old URLs would start redirecting to the new ones. Hope that helps.

**Reply by sylikc** [reply]
Ok, I enabled it just for a project... it wasn't entirely clear until reading that page again.  thanks
